configure.ac

```diff
@@ -306,6 +306,13 @@ EOF
   AC_MSG_RESULT($have_requests)
 ])
 
+# ShellCheck
+vmin_shellcheck=0.6.0
+AC_CHECK_PROG([SHELLCHECK], [shellcheck], [shellcheck])
+# https://stackoverflow.com/questions/6022384
+CH_CHECK_VERSION([SHELLCHECK], [$vmin_shellcheck],
+                 [--version | sed -n '2{s/^version: //;p;q}'])
```

Comment:
Can you explain this sed command?

Reply:
Yes; all the `CH_CHECK_VERSION` calls have something similar, to pull out a stand-alone version number without all the clutter from bare `--version`. If you're asking what the `-n` and `p;q` does, that *should* be explained in the SO link but I can clarify if needed.

Reply:
Ah. Perfect!